lib/Backends/CPU/FunctionSpecializer.cpp

```diff
@@ -151,6 +151,30 @@ class FunctionSpecializer {
     return specializedF;
   }
 
+  /// \returns true if a function is eligable for specialization.
+  bool isEligableForSpecialization(const llvm::CallInst *call) {
```

Comment:
May be. But it is unrelated to this PR. This function was not changed at all by the commits. But the function after it ('specializeCall') was moved from the private section of the class definition into a public section, because it is called from the newly introduced `specializeCallWithConstantArguments`.

But git for some reason does not quite understand what was moved ;-)

Reply:
nit: Eligable -> `Eligible`

Reply:
Good catch! Fixed.